akka-testkit-typed/src/test/scala/akka/testkit/typed/scaladsl/ManualTimerExampleSpec.scala

```diff
@@ -2,7 +2,6 @@ package akka.testkit.typed.scaladsl
 
 //#manual-scheduling-simple
 import akka.actor.typed.scaladsl.Behaviors
-import org.scalatest.WordSpecLike
```

Comment:
Another one for the docs

Reply:
Thanks for the correction! I pushed the rollback.